src/components/EmptyList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+import HelloIcon from './HelloIcon';
+import Navigation from './Navigation';
+
+const EmptyList = () => (
```

Comment:
Love this prompt! So intuitive and it really directs the user so that there's no doubt about what should be done once they've first created their list. So good. 🙌